packages/fhir-router/src/graphql.ts

```diff
@@ -703,6 +829,79 @@ async function resolveField(source: any, args: any, _ctx: GraphQLContext, info:
   return array;
 }
 
+/**
+ * GraphQL resolver function for create requests.
+ * The field name should end with "Create" (i.e., "PatientCreate" for updating a Patient).
+ * The args should include the data to be created for the specified resource type.
+ * @param _source The source/root object. In the case of creates, this is typically not used and is thus ignored.
+ * @param args The GraphQL arguments, containing the new data for the resource.
+ * @param ctx The GraphQL context. This includes the repository where resources are stored.
+ * @param info The GraphQL resolve info. This includes the schema, field details, and other query-specific information.
+ * @returns A Promise that resolves to the created resource, or undefined if the resource could not be found or updated.
+ */
+async function resolveByCreate(
+  _source: any,
+  args: Record<string, any>,
+  ctx: GraphQLContext,
+  info: GraphQLResolveInfo
+): Promise<Resource | undefined> {
+  const fieldName = info.fieldName;
+  const resourceType = fieldName.substring(0, fieldName.length - 'Update'.length) as ResourceType;
```

Comment:
`'Update'.length === 6`

`'Create'.length === 6`

It works, but probably not what you meant ;)